<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordFrm</class>
 <widget class="QWidget" name="RecordFrm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>提醒信息</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>491</width>
     <height>211</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::East</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>当次提醒</string>
    </attribute>
    <widget class="QRadioButton" name="ovRadio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>定时提醒</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTimeEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>194</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="odRadio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>延时提醒</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timeEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>118</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="osRadio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>下次启动时</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="ooRadio">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>下次连入互联网时</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="ofRadio">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>160</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>下次断开互联网时</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>规律提醒</string>
    </attribute>
    <widget class="QRadioButton" name="ryRadio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>每年</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rmRadio">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>每月</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rwRadio">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>每星期</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rdRadio">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>每天</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rhRadio">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>每时</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rmiRadio">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>每分</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="riRadio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>每隔</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rsRadio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>每次启动时</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="roRadio">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>180</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>每次连入互联网时</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rfRadio">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>180</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>每次断开互联网时</string>
     </property>
    </widget>
    <widget class="QComboBox" name="rcdCombo">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>公历</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>农历</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="rmCombo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="rdGroupBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="rddRadio">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="rdCombo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="rdwRadio">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="rwCombo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>第一个</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>第二个</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>第三个</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>第四个</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>第五个</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="rwdCombo">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>星期日</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>星期一</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>星期二</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>星期三</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>星期四</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>星期五</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>星期六</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QSpinBox" name="rvhSpin">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>时</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="rvmiSpin">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>30</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>分</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>446</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSpinBox" name="rySpin"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>年</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="rmSpin"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>月</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="rwSpin"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>星期</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="rdSpin"/>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>天</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="rhSpin"/>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>时</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="rmiSpin"/>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>分</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>491</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>提醒方式</string>
   </property>
   <widget class="QCheckBox" name="popupCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>弹出窗口</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="plainText">
     <string>窗口提示文字</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="soundCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>播放声音</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="soundGroupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>441</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="defaultSoundRadio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>89</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>默认声音</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="specSoundRadio">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>指定声音</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="soundEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="soundButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="autoturnCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>自动切换</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="runcmdCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>运行指令</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="delCheck">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>删除提醒</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>    此处输入“备注”名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cmdEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="addCheck">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>增加提醒</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="trayCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>托盘消息</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>备注：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>540</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>540</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>540</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
